src/main/steps/applicant1/review-the-application/template.njk

```diff
@@ -60,9 +60,20 @@
     <h3 class="govuk-heading-m">{{ subHeading10 }}</h3>
     <p class="govuk-body">{{ line22 }}</p>
   {% endif %}
-  <h3 class="govuk-heading-m">{{ subHeading11 }}</h3>
+  {% if isApplicant2Represented %}
+    <h3 class="govuk-heading-m">{{ subHeading11 }}</h3>
+    {% if not solInfoEntered %}
```

Comment:
why not flip them to avoid negation

Reply:
i believe i originally had that but i forgot why i flipped it so ill just go with your opinion.